profile/README.md

```diff
@@ -0,0 +1,41 @@
+# Polyxia 🚀
+
+- *[Polyxia](#polyxia---)*
```

Comment:
Useless ?

Reply:
the TOC or juste tu title ?

Reply:
`- *[Polyxia](#polyxia---)*`

this line